examples/tutorials/basics/polygons.py

```diff
@@ -0,0 +1,106 @@
+"""
+Plotting polygons
+=================
+
+Plotting polygons is handled by the :meth:`pygmt.Figure.plot` method.
+
+This tutorial focuses on input data given via lists or NumPy arrays. For
+plotting GeoPandas polygon geometry, e.g. to create a chorophlet map, see the
+gallery example :doc:`Choropleth map </gallery/symbols/maps/choropleth_map>`.
+"""
+
+# %%
+import numpy as np
+import pygmt
+
+# %%
+# Plot polygons
+# -------------
+#
+# Set up sample data points as lists for the x and y values.
+
+x_list = [-2, 1, 3, 0, -4, -2]
+y_list = [-3, -1, 1, 3, 2, -3]
+
+# %%
+# Create a Cartesian plot via the :meth:`pygmt.Figure.basemap` method.
+# Pass lists to the ``x`` and ``y`` parameters of the :meth:`pygmt.Figure.plot`
+# method. Without further adjustments, lines are drawn between the data points.
+# By default, the lines are 0.25-points thick, black, and solid. In this example,
+# the data points are chosen to make the lines form a polygon.
+
+fig = pygmt.Figure()
+fig.basemap(region=[-5, 5] * 2, projection="X5c", frame=True)
```

Comment:
I prefer to this version.
```suggestion
fig.basemap(region=[-5, 5, 5, 5], projection="X5c", frame=True)
```